packages/variable-editor/src/components/variables/data/metadata.test.ts

```diff
@@ -54,6 +57,44 @@ describe('isMetadataType', () => {
   });
 });
 
+describe('isMetadata', () => {
+  describe('true', () => {
+    test('empty', () => {
```

Comment:
I know testing ideas or flavours are different.
When I would have written this test, I would have created one test method with every thing in it, e.g. like:


```suggestion
test('isMetadata', () => {
  expect(isMetadata({ type: '' })).toBeTruthy();
  expect(isMetadata({ type: 'password' })).toBeTruthy();
  expect(isMetadata({ type: 'daytime' })).toBeTruthy();
  ...
  expect(isMetadata({ type: 'other' as MetadataType })).toBeFalsy();
  ...
});
```

Reply:
I like it better the current way. If something fails, you will immediately know which one did. Are there other advantages/disadvantages?

Reply:
if the fail message is good enought I have all the info which I need ;)
Less code and IMO a better overview. 
But as I said this my opinion. I would not say that your way is wrong :)